src/openscad.cc

```diff
@@ -920,6 +920,7 @@ int main(int argc, char **argv)
 		("d,d", po::value<string>(), "deps_file -generate a dependency file for make")
 		("m,m", po::value<string>(), "make_cmd -runs make_cmd file if file is missing")
 		("quiet,q", "quiet mode (don't print anything *except* errors)")
+		("parametercheckoff", "disables the parameter check")
```

Comment:
good idea. we could also add an option to disable all optional warnings.

Yes this would be the first optional warning and I assume that we will add more for legacy support.

`-Wundeclared-parameter` is awkward because of the capital letter and that we normally use `-` for the short parameter name and `--` for the long name.

I will dwell a bit on that.

Reply:
I borrowed the syntax from gcc, but we could come up with whatever fits our current style.